ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -319,25 +327,25 @@ tests:
       OCP_ARCH: arm64
       WORKER_INSTANCE_TYPE: m6g.xlarge
     workflow: openshift-e2e-aws-upi
-- as: ocp-e2e-aws-ovn-arm64-single-node
+- as: ocp-e2e-aws-ovn-sno-multi-a-a
```

Comment:
Here's another strange naming convention:

[TEST SUITE]-[PLATFORM]-[PAYLOAD]-[DEPLOYMENT STYLE]-[CONTROL PLANE ARCH]-[COMPUTE NODE ARCHITECTURE(S)]

test suite = ocp-e2e
platform = aws
missing network type? = ovn
deployment type = sno
payload = multi
ctlp = a
cmp = a

Deployment and payload appear to be reversed.